src/main/java/org/tron/core/Constant.java

```diff
@@ -47,15 +47,12 @@
   public static final long TRANSACTION_DEFAULT_EXPIRATION_TIME = 60 * 1_000L; //60 seconds
 
   // config for smart contract
-  public static final long MEM_LIMIT_IN_ONE_TX_OF_SMART_CONTRACT = 32 * 1024 * 1024L; // 32MB
-  public static final long MAX_CPU_TIME_OF_ONE_TX = 50000; // 50 ms = 50000 us
-  public static final long STORAGE_LIMIT_IN_ONE_TX_OF_SMART_CONTRACT = 32 * 1024 * 1024L; // 32MB
   public static final long SUN_PER_ENERGY = 100; // 1 us = 100 DROP = 100 * 10^-6 TRX
-  public static final long MAX_ENERGY_IN_TX = 3000000; // ref: 1 us = 1 energy
+  public static final long ENERGY_LIMIT_IN_CONSTANT_TX = 3000000; // ref: 1 us = 1 energy

```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add underscores to this numeric value for readability ([squid:S2148](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2148))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1670&issues=AWbTWpBiLpqqSUlcalJs&open=AWbTWpBiLpqqSUlcalJs)

[ISSUE_KEY]: AWbTWpBiLpqqSUlcalJs